Makefile

```diff
@@ -98,6 +98,7 @@ dockerized-test:
 	  -e ES_URL=http://elasticsearch:9200 \
 	  -e KIBANA_URL=http://kibana:5601 \
 	  -e DJANGO_URL="http://djangoapp:8003" \
+	  -e DOTNET_URL="http://dotnetapp:80" \
```

Comment:
Let's review whether it's the internal or external port, if the second one then: https://github.com/elastic/apm-integration-testing/pull/424/files#diff-d0df5dd5f0019c3b878fde30f4b7341bR1356

Reply:
```
docker run  --name=apm-integration-testing1  --network=apm-integration-testing  --security-opt seccomp=unconfined  --rm  --entrypoint '' -ti apm-integration-testing /bin/bash
tester@daf2b7882782:~$ curl http://dotnetapp:80/
OKtester@daf2b7882782:~$ curl http://dotnetapp:8100
curl: (7) Failed to connect to dotnetapp port 8100: Connection refused
```